guides/source/active_record_callbacks.md

```diff
@@ -670,7 +1007,8 @@ User was saved to database
 
 ### `after_save_commit`
 
-There is also [`after_save_commit`][], which is an alias for using the `after_commit` callback for both create and update together:
+There is also [`after_save_commit`][], which is an alias for using the
+`after_commit` callback for both create and update together:
 
 ```ruby
 class User < ApplicationRecord
```

Comment:
Let's update this example to remove the `@` instance var in favor of just a local var too.

Reply:
✅